VK/VK/SearchGroupScreen/SearchGroupTableViewController.swift

```diff
@@ -11,6 +11,12 @@ final class SearchGroupTableViewController: UITableViewController {
     var hidenGroups: [NetworkUnit] = []
     var searchedGroup: NetworkUnit?
```

Comment:
```suggestion
    var searchedGroup: NetworkUnit?
```
не совпадают названия